tests/python/unittest/test_operator.py

```diff
@@ -5444,6 +5444,21 @@ def get_output_names_callback(name, arr):
     lrn_sym = mx.sym.LRN(data, nsize=1, name='lrn')
     check_name(lrn_sym, ['lrn_output', 'lrn_tmp_norm'])
 
+    act_sym = mx.sym.Activation(data, act_type='relu', name='act')
```

Comment:
Please also add unit test for the change pooling output names. Need to cover the both cases of the number of outputs equal to 1 and 2.

Reply:
we can't. the case that pooling has two outputs is when mkldnn is enabled. we don't know when mkldnn is enabled in python.

Reply:
Shouldn't the additional output be invisible?

Reply:
@zheng-da Since the second output is invisible to users, you can use `FNumVisibleOutputs` to verify the first output name is "output" for the pooling op at least.

Reply:
I have added a test for pooling.